src/main/resources/templates/charsheet.html

```diff
@@ -39,57 +41,53 @@
         </div>
       </div>
 
+      <div class="section_devider"></div>
+
       <div class="section" id="mid">
         <div class="stat" id="level">
-          <span id="levelLabel" class="statLabel">LV</span>
-          <span id="levelValue" class="statValue" th:text="${character.level}" th:if="${!admin}">1</span>
+          <span id="levelLabel" class="statLabel"><p>LV</p></span>
+          <span id="levelValue" class="statValue" th:text="${character.level}" th:if="${!admin}"><p>1</p></span>
```

Comment:
These `th:text` fields (used for Thymeleaf templating) will break with your new nested elements, as will all of the JS that updates these fields as you spend skill points or as the admin adjusts levels, etc., etc.

If you move the `th:text` and the `id` attributes into the nested `p` element for each of them, they Should™ continue working.

Reply:
Alright good to know!